go/engine/device_history.go

```diff
@@ -132,6 +133,23 @@ func (e *DeviceHistory) loadDevices() error {
 		e.devices = append(e.devices, exp)
 	}
 
+	// Load the last used times, but only if these are your own devices. The
+	// API won't give you those times for other people's devices.
+	if e.user.GetNormalizedName().Eq(e.G().Env.GetUsername()) {
+		lastUsedTimes, err := e.getLastUsedTimes()
+		if err != nil {
+			return err
+		}
+		for i := range e.devices {
+			detail := &e.devices[i]
+			lastUsedTime, ok := lastUsedTimes[detail.Device.DeviceID]
+			if !ok {
+				return fmt.Errorf("Failed to load last used time for device %s", detail.Device.DeviceID)
```

Comment:
should it just keep going in this case?  is it possible to have a device that's never been used?